File: culturio-src/application/Users/Models/UserDto.cs
using Culturio.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Culturio.Application.Users.Models
{
    public class UserDto
    {
        public int Id { get; set; }
        public string ExternalId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Role { get; set; }

        public string? Phone { get; set; }
        public string? Address { get; set; }
        public int? PostalCode { get; set; }
        public string? City { get; set; }
        public string? Sex { get; set; }        
        public string? DefaultLanguage { get; set; }
        public bool? TermsAccepted { get; set; }
        public DateTime DateCreated { get; set; }
        public bool? IsActive { get; set; }
        public string? Company { get; set; }
        public string? CultureObject { get; set; }
        public string? QRcode { get; set; }
    }
}

File: culturio-src/web/users/models/userDto.ts
export class UserDto {
  public id: number;
  public firstName: string;
  public lastName: string;
  public address: string;
  public city: string;
  public email: string;
  public sexId: number;
  public roleId: number;
  public defaultLanguage: string;
  public termsAccepted: boolean;
  public postalCode: number;
  public phone: string;
  public qRcode: string;
  public isActive: boolean;
  public companyId: number;
  public cultureObjectId: number;
  constructor(
    id: number,
    firstName: string,
    lastName: string,
    address: string,
    city: string,
    email: string,
    sexId: number,
    roleId: number,
    defaultLanguage: string,
    termsAccepted: boolean,
    postalCode: number,
    phone: string,
    qRcode: string,
    isActive: boolean,
    companyId: number,
    cultureObjectId: number
  ) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.address = address;
    this.city = city;
    this.email = email;
    this.sexId = sexId;
    this.roleId = roleId;
    this.defaultLanguage = defaultLanguage;
    this.termsAccepted = termsAccepted;
    this.postalCode = postalCode;
    this.phone = phone;
    this.qRcode = qRcode;
    this.isActive = isActive;
    this.companyId = companyId;
    this.cultureObjectId = cultureObjectId;
  }
}
